### ISSUE TITLE
[SP] Issue with gregtech pipes not loading correctly

### ISSUE THREAD

**Issue by Adz-Scott** [GTNewHorizons/NewHorizons#1294]
Apologies - browser crashed in the middle of typing the description.

I had a set up involving multiple water-towers leading to a steel boiler multiblock which ran for 8-9 hours of gameplay yesterday without issue, however when I loaded up the same world this evening it took less than two minutes for the boiler to run dry and explode.
Upon inspection it appears that the shutter-covers I put on several pipes all reset to their default state when the client was reloaded, thus causing the flow or water to be disrupted.

tl;dr  shutter-covers reset to default state when client reset. Things go boom.

**Comment by Dream-Master** [comment]
@Adz-Scott a bit more ifo would be helpfull.

**Comment by Dream-Master** [comment]
@Adz-Scott Not get new information on this so i close it now.
